### PR TITLE
Remove PreviousVersion attributes for previous milestones

### PR DESCRIPTION
**Pull request by BHoMBot**
Remove PreviousVersion attributes for previous milestones

### CODE REVIEW

**Comment by FraserGreenroyd** [comment]
@BHoMBot check required

File: Graphics_Engine/Modify/CenterGradientAsymmetric.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿/*
+/*
```
Comment (IsakNaslundBh) [inline Graphics_Engine/Modify/CenterGradientAsymmetric.cs:1]: Not sure what has happened here on git? File on my machine on master does not have a `}` at the top of the file. Probably fine though?

**Review by IsakNaslundBh — APPROVED** [review]
Looks good to me!

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

**Comment by FraserGreenroyd** [comment]
@BHoMBot check installer
@BHoMBot check core
@BHoMBot check compliance
@BHoMBot check null-handling
@BHoMBot check serialisation

**Comment by FraserGreenroyd** [comment]
@BHoMBot check ready-to-merge

**Comment by FraserGreenroyd** [comment]
@BHoMBot this is a CI/CD instruction. I am authorising dispensation to be granted on check ref. 2997424563

**Pull request merged**